GA-worksheet.md

```diff
@@ -75,21 +81,21 @@
 
 6. What does each digit of this Hexidecimal Color Code represent? #E78
 
- E represents ...
+ E represents ... the amount of redness.
 
- 7 represents ...
+ 7 represents ... the amount of greenness.
 
- 8 represents ...
+ 8 represents ... the amount of blueness.
 
-7. What is the Hex Color Code for true purple?  #
+7. What is the Hex Color Code for true purple?  #509
 
-8. _______________ is the programming language that allows websites to be interactive.
+8. Javascript is the programming language that allows websites to be interactive.
 
-9. This tag allows us to add JavaScript into any webpage? <          >
+9. This tag allows us to add JavaScript into any webpage? <script>
 
-10. What is an event?
+10. What is an event? An event is an alert informing the user something. 
```

Comment:
An event is basically anything that happens on the web page: click, hover, page load. We used the alert in this tutorial to alert us of a "click" event.